### ISSUE TITLE
Bounds missing for textLayer objecs

### ISSUE THREAD

**Issue by xorb** [opendesigndev/open-design-sdk#23]
Bounds property missing for Adobe Illustrator text layers.
```
{
          id: "0:48",
          visible: true,
          type: "textLayer",
          opacity: 1,
          text: {
            transformMatrix: { a: 0.7227, b: -0.6911, c: 0.6911, d: 0.7227, tx: 226.2002, ty: 358.3203 },
            value: "O",
            styles: [
              {
                color: { r: 191, g: 234, b: 235, a: 1 },
                font: { name: "DIN Next LT Pro Condensed", size: 8, postScriptName: "DINNextLTPro-Condensed", bold: false, italic: false, letterSpacing: 0 },
                ranges: [{ from: 0, to: 1 }],
              },
            ],
          },
          name: "O",
          origin: "illustrator",
}
```

Also, this is the reason why exportToSVG is not working for AI files with text elements.

**Comment by rhefner1** [comment]
Hey @xorb , sorry for the radio silence on this. @petrbrzek @rychlis can you investigate here?

**Comment by rhefner1** [comment]
Hey @xorb , I heard back from the team on this. Exporting Illustrator text layers as SVG isn't fully supported right now since we don't get the text layer bounds from the Illustrator file format. In the future, we'd like to be able to "guess" the bounds using our rendering engine so that this would work, but it isn't ready right now. As a workaround, you can also export this text layer as a PNG using Render.

If you need to get the position, you can use the `text.transformMatrix` property to figure out where the layer is positioned.

**Comment by xorb** [comment]
@rhefner1 can you provide some details about how I can use transformMatrix ? Have tried, but not able to get accurate positions.

**Comment by janroztocil** [comment]
Fixed in `0.4.5`.

As @rhefner1 said, we don't know bounds of text layers from Illustrator therefore they are are not present in Octopus. However, bounds can be obtained via `getBounds` call https://opendesign.dev/docs/sdk-reference/layerfacade#get-bounds (this has been fixed in the latest update).

**Comment by rhefner1** [comment]
@xorb let me know if this fix works for you!
